pkg/entity/entity.go

```diff
@@ -449,7 +449,8 @@ func (w Workspace) GetProjectFolderPath() string {
 	if MapContainsKey(LegacyWorkspaceGroups, w.WorkspaceGroupID) {
 		prefix = "/home/brev/workspace"
 	} else {
-		prefix = "/home/ubuntu"
+		sshUser := w.GetSSHUser()
```

Comment:
It may be worth returning an error if this is empty, just to make the possible support case that much clearer.

Quick check of the database makes it look like this should be safe though -- the only time when the ssh user is empty, from what I can see, is when the instance is not in a running state.